### PR TITLE
mpOT 1.3: Rest Client support

### LINKED ISSUE #6665: Tracing support in MP Rest Client
Implementation of MP OpenTracing 1.3

### PR DESCRIPTION
**Pull request by fmhwong**
Fixes #6665 

1. Add `org.eclipse.microprofile.opentracing:microprofile-opentracing-api:1.3` to oss dependency
1. Add `opentracing-1.3` and `mpOpenTracing-1.3` and the OT-RestClient auto feature
1. Add `com.ibm.websphere.appserver.spi.opentracing.1.3`
1. Add `com.ibm.websphere.org.eclipse.microprofile.opentracing.1.3`
1. Add `com.ibm.ws.microprofile.opentracing.1.3`
1. Add `com.ibm.ws.microprofile.opentracing.rest.client.1.3`
1. Add `com.ibm.ws.opentracing.1.3.cdi`
1. Add `com.ibm.ws.opentracing.1.3`
1. Add `com.ibm.ws.opentracing.1.3_fat`

### CODE REVIEW

**Comment by fmhwong** [comment]
#build

**Comment by fmhwong** [comment]
#build

File: dev/com.ibm.ws.opentracing.1.3.cdi/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.2
```
Comment (tonyreigns) [inline dev/com.ibm.ws.opentracing.1.3.cdi/bnd.bnd]: Should be version 1.3

**Comment by fmhwong** [comment]
The one error from PB is a re-occurrence of https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=246081 and it is not related to the PR.

**Pull request merged**